.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# These owners will be the default owners for everything in
+# the repository and will be requested for review.
+
+# Curriculum Development team.
+*       @jtascensao @PedroGFonseca
+
+# Add Specialization Leads.
```

Comment:
Is all the discussion about curriculum going to happen in this repository?

If so, just to make sure that I completely understood this, we're going to have different directories in this repo, each with all the relevant files for the bootcamp and each specialisation. Each of this directories will have its own codeowners and in this way, we don't need to have the specialisation lead of spec A being code owner of the other specialisations. Is this right?

Reply:
Yes, it is!